index.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Kmarketer</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles/index.css">
+</head>
+
+<body>
+  <header class="header">
+    <h3 class="logo">Kmarketer</h3>
+    <nav class="header__left-nav">
+      <nav class="header__nav-links">
+        <a href="#" class="headline">Home</a>
+        <a href="#" class="headline">Product</a>
+        <a href="#" class="headline">Pricing</a>
+        <a href="#" class="headline">Contact</a>
+        <img class="header__nav-icon" src="./assets/icons/icn search .icn-xs.svg" alt="search icon">
+        <img class="header__nav-icon" src="./assets/icons/icn shopping-cart .icn-xs.svg" alt="cart icon">
+      </nav>
+      <img class="header__toggler" src="./assets/images/navbar-toggler-icon.png" alt="toggler hambuger icon">
+    </nav>
+  </header>
+
+  <section class="hero">
+    <div class="section-text section-text--long-gap section-text--centered">
+      <h1 class="section-text__heading section-text__heading--big">Experts are here solve
+        your business problem.</h1>
+      <p class="description description--big">We know how large objects will act, but things
+        on a
+        small scale just do not act that way.</p>
+    </div>
+    <div class="hero__buttons-container">
+      <button type="button" class="button button--primary button--rounded">Get Quote Now</button>
+      <button type="button" class="button button--secondary button--rounded">Learn more</button>
+    </div>
+  </section>
+
+  <section class="features">
+    <div class="section-text">
+      <h2 class="section-text__heading">WHY CHOOSE US</h2>
+      <p class="description">Problems trying to resolve the conflict between
+        the two major realms of Classical physics: Newtonian mechanics</p>
+    </div>
+    <div class="features__card-container">
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/icn settings .icn-lg.svg" alt="briefcase icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="feature-card__description">the quick fox jumps over the lazy
```

Comment:
I suggest here you reuse descriptive class with modifiers instead of creating new element.